meetings/2024-10-23.md

```diff
@@ -0,0 +1,154 @@
+* Jesse: I know there are some questions left unanswered, I'll answer those later in the matrix channel or on the forum.
+* Tim: And it needs funding, we've applied to various people, for example people who previously funded the ODI. One of the things the Solid community can do is to try to raise the total incoming amount of grants, for example with people like NLnet.
+
+### Farming C2C
+
+Maxime: At DFC we are working on c2c specs about farming but in fact it's even more large -> ERP systems. I would like to propose some work item next.
+Michiel: Yes, I think you can just create a PR on [the work items list](https://github.com/solid/specification/blob/df7cee6c4225b9cab5eedba3aec51074aeb7c067/index.html#L122)
+ 
+ TBL: Shall we discuss it today.
+ Maxime: Not today
+
+
+### Error Notifications
+
+ 
+
+https://github.com/solid/notifications/issues/198
+
+* RG: We need a unified pattern for notifying notification errors.
+* RG: You need to be able to communicate when there is an error between the resource server and the notficiation centre. You need to communicate that to the client, should be RDF, some discussion on GH, but wanted to bump here so that anyone with an opinion can chime in.
+* Elf: We need some more specific use cases and requirments from the perspective of the client. We need to arrive at the interface we want based on who consumes it. We have already arrived at a general interface based on the fact that error was unreliable, for instance may know that resource needs to be re-requested if the error was unreliable. This is because the notification system knows that it could have missed some updates. If you know that there are some scenarios where you would like an error to happen, then identify what you would have like to have happened.
+ * RG: I've begun that process by putting what those generic requirements might look like in terms of classes of products.
+ * Elf: I've been doing webhooks in the last few months, if we are doing errors then it impacts the notification recievers as they need to check if at as error message etc if errors are included with notifications.
+
+### Activity Mapping for Notifications
+
+https://github.com/solid/notifications/issues/183
+
+* RG: Added a new column; need to revisit this. Looking at what are the container mappings, because this is something we need to revisit soon, the purpose of this issue was to have an [?? office hours STM??]
+* eP: Should not map to HTTP Headers. Solid or LWS should define set of operations and notification system hooks into set of operations. Then there will be HTTP interface that is binding this to HTTP methods but notif. system should not bind directly to that. Relevant to discuss in relation to access control and notif system, both should have a common layer of internal operations supported by the storage. That is something LWS may need to take care of by making those operations defined
```

Comment:
```suggestion
* RG: Added a new column; need to revisit this. Looking at the container mappings, because this is something we need to revisit soon. The purpose of this issue was to have an [?? office hours STM??]
* eP: Should not map to HTTP Headers. Solid or LWS should define a set of operations, and the notification system hooks into the set of operations. Then there will be an HTTP interface that binds this to HTTP methods by notifications. System should not bind directly to that. Relevant to discuss in relation to access control and notification system; both should have a common layer of internal operations supported by the storage. That is something LWS may need to take care of by defining those operations.
```